### ISSUE TITLE
Standby Cluster setup issue

### ISSUE THREAD

**Issue by dalton007-1** [zalando/patroni#1236]
I'm working on setting up a standby cluster.  My primary cluster setup consists of 3 VMS running postgres w/postgis, pgbouncer, patroni, vip-manager; 5 VMs running ETCD. I have verified this setup works correctly. DB failover, vip-manager IP reassignment, so forth.  VIP manager virtual IP for primary cluster is 192.168.123.201. I use port 6432 to connect since I'm using pgbouncer.

I have setup another second cluster on another physical machine. Same VM setups as my primary. I'm now attempting to turn that into a standby cluster. For the second cluster, I started with a fresh working cluster as outlined above, tested as outlined above, then added the following using 'patronictl edit-config':

standby_cluster:
  host: 192.168.123.201
  port: 6432
  primary_slot_name: patroni
  replica_methods:
  - basebackup

To point it towards my primary cluster. I commit this change & restart patroni service on all PGIS VM's. Then using journalctl -xe I see the following error message:

Oct 18 18:14:04 afesX2-pgis01 patroni[1190]: 2019-10-18 18:14:04,168 ERROR: Exception when working with leader
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]: Traceback (most recent call last):
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:   File "/usr/local/lib/python3.6/dist-packages/patroni/postgresql/rewind.py", 
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:     with get_connection_cursor(connect_timeout=3, options='-c statement_timeou
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:   File "/usr/lib/python3.6/contextlib.py", line 81, in __enter__
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:     return next(self.gen)
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:   File "/usr/local/lib/python3.6/dist-packages/patroni/postgresql/connection.p
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:     with psycopg2.connect(**kwargs) as conn:
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:   File "/usr/local/lib/python3.6/dist-packages/psycopg2/__init__.py", line 126
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]:     conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]: psycopg2.OperationalError: ERROR:  unsupported startup parameter: options
Oct 18 18:14:04 afesX2-pgis01 patroni[1190]: 2019-10-18 18:14:04,175 INFO: no action.  i am the standby leader with the loc

This is my second try with cluster 2-that is I got the error the first time, tried fresh VMs/setting everything up again. patronictl list shows all machines up on both clusters. I am using the same cluster name for both the primary and secondary cluster, did not think that would be the issue.

Any advice/pointers? Need more info? My goal is to setup a secondary cluster with a 24 hour delay but I though I'd get the secondary up & running first then try to setup the delay.

**Comment by CyberDem0n** [comment]
It won't work via pgbouncer

**Comment by dalton007-1** [comment]
OK thanks for the quick response. I will try testing without pgbouncer.

**Comment by dalton007-1** [comment]
I have started with fresh clusters; same config as originally listed, not using pgbouncer. I also added each member of the secondary cluster to the pg_hba section using patroni_ctl.  I'm now seeing the following error:

Oct 25 17:59:02 afesx2-pgis02 patroni[1266]: 2019-10-25 17:59:02.170 UTC [1741] FATAL:  database system identifier differs between  the primary and standby
Oct 25 17:59:02 afesx2-pgis02 patroni[1266]: 2019-10-25 17:59:02.170 UTC [1741] DETAIL:  The primary's identifier is 6750688166659451152, the standby's identifier is 6751767541379033136.
